Documentation/CONFIG.md

```diff
@@ -1,6 +1,6 @@
 # MetroAE Config
 
-MetroAE Config is a template-driven VSD configuration tool. It utilizes the VSD API along with a set of common Feature Templates to create configuration in the VSD. You will create a yaml or json data file with the necessary configuration parameters for the particular feature and execute simple CLI commands to push the configuration into VSD. You can find out more about MetroAE Config by referring to [MetroAE Config Overview](CONFIG_OVERVIEW.md).
+MetroAE Config is a template-driven VSD configuration tool. It utilizes the VSD API along with a set of common Feature Templates to create configuration in the VSD. You will create a yaml or excel spreadsheets (.xlsx files) with the necessary configuration parameters for the particular feature and execute simple CLI commands to push the configuration into VSD. You can find out more about MetroAE Config by referring to [MetroAE Config Overview](CONFIG_OVERVIEW.md).
```

Comment:
For consistency, capitalize Excel here and leave it singular (Excel spreadsheet (.xlsx file))